workflows/testing/test_workflow/integ_test.py

```diff
@@ -0,0 +1,13 @@
+import os
+
+class IntegTestSuite:
+    def __init__(self, name, repo):
+        self._name = name
+        self._repo = repo
+
+    def execute(self, cluster):
+        script = self._repo.dir() + "/integtest.sh"
+        if (os.path.exists(script)):
+            repo.execute(f'sh integtest.sh -b {cluster.endpoint()} -e {cluster.port()}')
```

Comment:
-e?

Reply:
This doesn't fail when integ tests fail.

Reply:
Yup, it's a prototype. Fault tolerance will come if/when we turn this into a production system.